gatk.wdl

```diff
@@ -1401,3 +1458,62 @@ task SplitNCigarReads {
                       category: "advanced"}
     }
 }
+
+task VariantFiltration {
+    input {
+        File inputVcf
+        File inputVcfIndex
+        File referenceFasta
+        File referenceFastaDict
+        File referenceFastaFai
+        String outputPath = "filtered.vcf.gz"
+        Array[String]+ filterArguments
```

Comment:
Hmm, I don't really like this. Is this always a combination of `filter_name` and `filter_expression`, maybe in that case a Pair could be used in some way?

Reply:
On the command line you would need to do this: `--filter-name my_filter --filter-expression 'GQ<30' --filter-name my_other_filter --filter-expression 'AD>2.0'`

So I reckoned that a list is the easiest way: `["--filter-name", "my_filter", "--filter-expression", "'GQ<30'", "--filter-name", "my_other_filter", "--filter-expression", "'AD>2.0'"]`

But maybe a string is even easier:
`"--filter-name my_filter --filter-expression 'GQ<30' --filter-name my_other_filter --filter-expression 'AD>2.0'"`

Pairs are difficult to deserialize in tasks + they are a pain to use as input in JSON. This is a list of pairs... 

What do you think?